src/main/java/nextstep/subway/domain/LineStation/LineStationRepository.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.subway.domain.LineStation;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface LineStationRepository extends JpaRepository<LineStation, Long> {
+
+    List<LineStation> findByStationId(Long stationId);
+}
```

Comment:
```suggestion
}

```

- 여러 버전의 유닉스 OS간의 공통된 API 및 인터페이스를 정의해놓은 POSIX 명세 중 text file 부분을 보면 유닉스 시스템은 text file들을 서로 구분할때 EOL을 사용하기 때문에 파일을 만들때마다 파일의 끝에 EOL을 넣어주는 것이 안전하다.

EOL을 추가해보면 어떨까요?

Reply:
오.. 처음 알았습니다. 클래스 맨 아래에 개행 문자를 추가해주면 되는거죠?

Reply:
맞습니다~